server/postgreSchema.sql

```diff
@@ -0,0 +1,23 @@
+CREATE SCHEMA ReservationsModule;
+
+CREATE TABLE ReservationsModule.Restaurants (
+  id int not null unique,
+  name str,
+  open time,
+  close time,
+  num_seats int
+)
+
+CREATE TABLE ReservationsModule.Reservations (
+  id int not null unique,
+  restaurant_id int,
+  user_id int,
+  date date,
+  time time,
```

Comment:
If you need both Date and Time values, you can use TIMESTAMP which stores both date and time.